docs/proposals/network-policy.md

```diff
@@ -197,6 +254,35 @@ type NetworkPolicyPort struct {
 	Port *intstr.IntOrString `json:"port,omitempty"`
 }
 
+type NetworkCIDRRange struct {
+	// QUESTION: we could just do "Start" and "End" without a prefix to
+	// specify the range.  While this provides a simpler structure, it may
+	// be more error-prone since its less easily validated, and isn't very
+	// nice for IPv6 addresses.
```

Comment:
> Basically, Prefix makes sense if you want to deny traffic by default and only allow to a few networks, but not if you want to allow by default and deny to a few networks

I think the problem here is that we don't do "allow by default, then deny using policy" or any combination of "allow / deny".  I'm worried that the start/prefix/end notation is basically a workaround to avoid us implementing proper deny.